docs/manifest.json

```diff
@@ -4456,6 +4456,14 @@
                 "main"
             ]
         },
+        {
+            "id": "cloud-vmware-engine",
+            "name": "google/cloud-vmware-engine",
+            "defaultService": "vmwareengine/readme",
+            "versions": [
+                "main"
+            ]
+        },
```

Comment:
We can stop making changes to the `docs/` directory since these are no longer used. I'll submit a follow-up PR to remove this, but you may want to update your script!